windows/ReactNativeWebView/ReactWebViewManager.cpp

```diff
@@ -40,13 +40,17 @@ namespace winrt::ReactNativeWebView::implementation {
     IMapView<hstring, ViewManagerPropertyType> ReactWebViewManager::NativeProps() noexcept {
         auto nativeProps = winrt::single_threaded_map<hstring, ViewManagerPropertyType>();
         nativeProps.Insert(L"source", ViewManagerPropertyType::Map);
+        nativeProps.Insert(L"backgroundColor", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"messagingEnabled", ViewManagerPropertyType::Boolean);
         return nativeProps.GetView();
     }
 
     void ReactWebViewManager::UpdateProperties(
         FrameworkElement const& view,
         IJSValueReader const& propertyMapReader) noexcept {
-        if (auto webView = view.try_as<winrt::WebView>()) {
+        auto control = view.try_as<winrt::UserControl>();
```

Comment:
If we're going to access after this, `winrt::as` might be appropriate. The difference is that it's fail-fast vs returning null.

Reply:
Same below

Reply:
7cb13c796af5105d5ed461337454cc6fa46170cd